samples/Sentry.Samples.Maui/MainPage.xaml.cs

```diff
@@ -73,7 +73,7 @@ private void OnJavaCrashClicked(object sender, EventArgs e)
 
     private void OnNativeCrashClicked(object sender, EventArgs e)
     {
-#if ANDROID || IOS || MACCATALYST
+#if __MOBILE__
```

Comment:
worth adding a `#else` with a log Info "hey this doesn't do anything here"

Reply:
I already make the button invisible, so no need really.